frame/multisig/src/lib.rs

```diff
@@ -280,10 +272,6 @@ decl_module! {
 			call: Box<<T as Trait>::Call>,
 		) -> DispatchResultWithPostInfo {
 			let who = ensure_signed(origin)?;
-			// We're now executing as a freshly authenticated new account, so the previous call
-			// restrictions no longer apply.
-			let _guard = ClearFilterGuard::<T::IsCallable, <T as Trait>::Call>::new();
-			ensure!(T::IsCallable::filter(call.as_ref()), Error::<T>::Uncallable);
```

Comment:
Something is needed to replace these lines - the local filter basically needs to be temporarily cleared.

Reply:
I think it is, in the code below, in the if branch and in the else branch, the call is dispatched with:
```
let result = call.dispatch(frame_system::RawOrigin::Signed(id.clone()).into());
```
thus origin for dispatched have BaseCallFilter, and doesn't inherit from given origin.

Reply:
ahh true.